src/player.cpp

```diff
@@ -6445,6 +6447,9 @@ void player::hardcoded_effects(effect &it)
             }
         }
     } else if( id == effect_bleed ) {
+        // Prolonged haemorrhage is a significant risk for developing anaemia
+        vitamin_mod( vitamin_iron, rng( -1, -4 ) );
```

Comment:
Would probably be better with the damage, below. That way it would scale with intensity.

Also, it looks really fast here. A bleeding that would cause 30 damage on one body part could easily drain 180 units of iron.